core/docs/API.md

```diff
@@ -21,32 +21,315 @@ The API exposes functions for:
 
 C API
```

Comment:
This is not needed. Instead, promote the Headings such as `State Management` or `Geometry` etc. to this level (i.e. with a line beneath.